apps/dashboard/src/components/product-pages/common/Topnav.tsx

```diff
@@ -1,43 +1,20 @@
-import { Box, Container, Flex, useBreakpointValue } from "@chakra-ui/react";
-import { useScrollPosition } from "@n8tb1t/use-scroll-position";
```

Comment:
This navbar is only used in the pages that (pretty sure) will be converted to Framer.

What it does is change the background color effects on scrolling down. It not that big of a deal and I think we can just make it "static".